ci/values-staging.yaml

```diff
@@ -3,3 +3,7 @@ nodeSelector:
 presets:
   logsCollection:
     enabled: false
+useGOMEMLIMIT: true
+resources:
+  limits:
+    memory: 160Mi
```

Comment:
Should we remove this, as this was just for testing purposes ?

Reply:
the `memory: 160Mi`, not the `useGOMEMLIMIT: true`

Reply:
Removed!